python/ml4ir/base/model/relevance_model.py

```diff
@@ -778,3 +780,33 @@ def _build_callback_hooks(
         # Add more here
 
         return callbacks_list
+
+    def calibrate(self, relevance_dataset, logger, logs_dir_local, temperature_init)\
```

Comment:
To me it is still striking that we are defining with to the base relevance model and we explicitly define a `NotImplemented` in the ranking side to bypass this. It suggests that this function is not generic enough for the base model and should live in the classification model @marjanhs @lastmansleeping .